packages/lfo-input/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Would be good to have a root package script to generate a new sub-package to add all the config files, etc. Maybe could even add it to the list desktop consumes.